### PR TITLE
RESTWS-792:removed unused code from CohortMemberResource1_8, Resource…

### PR DESCRIPTION
**Pull request by EllingtonK**
RESTWS-792: removed unused code from CohortMemberResource1_8, ResourceRepresentation, SwaggerSpecificationCreator, and RestUtil

https://issues.openmrs.org/browse/RESTWS-792

This was all deletes requested except RestUtil.java#L224-L413.  Comments in RestUtil.java file say lines need moved to a separate method.  Has this been done yet or is still needed?

### CODE REVIEW

File: omod-1.8/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_8/CohortMemberResource1_8.java [hunk 1]
```diff
@@ -267,25 +266,6 @@ protected String getUniqueId(CohortMember1_8 delegate) {
...
-	List<Object> ret = new ArrayList<Object>();
-	Representation rep = Representation.FULL;
-	if (context.getRepresentation().equals(Representation.DEFAULT))
-		rep = Representation.REF;
-	else if (context.getRepresentation().equals(Representation.FULL))
-		rep = Representation.DEFAULT;
-	for (CohortMember member : doGetAll(parent, context))
-		ret.add(asRepresentation(member, rep));
-	return ret;
-	}*/
```
Comment (sacull) [inline omod-1.8/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_8/CohortMemberResource1_8.java:269]:
```suggestion
```
We only use two blank lines when we have a good reason. ;)

File: omod-common/src/main/java/org/openmrs/module/webservices/docs/swagger/SwaggerSpecificationCreator.java [hunk 2]
```diff
@@ -794,7 +794,7 @@ private void addPaths() {
 			/////////////////////////
 			Path uuidPathDelete = buildDeletePath(uuidPathPostUpdate, resourceHandler, resourceName,
 			    resourceParentName);
-			//addIndividualPath(pathMap, uuidPathDelete, resourceParentName, resourceName, uuidPathDelete, "/{uuid}");
+
```
Comment (sacull) [inline omod-common/src/main/java/org/openmrs/module/webservices/docs/swagger/SwaggerSpecificationCreator.java:797]:
```suggestion
```
Same as above.

File: omod-common/src/main/java/org/openmrs/module/webservices/docs/swagger/SwaggerSpecificationCreator.java [hunk 3]
```diff
@@ -1039,7 +1039,7 @@ private Operation createOperation(DelegatingResourceHandler<?> resourceHandler,
 		
 		// create definition
 		if (operationName == "post" || operationName == "get") {
-			//			createDefinition(operationEnum, resourceName, resourceParentName, representation);
+
```
Comment (sacull) [inline omod-common/src/main/java/org/openmrs/module/webservices/docs/swagger/SwaggerSpecificationCreator.java:1042]:
```suggestion
```
What is the function of this blank line?

File: omod-common/src/main/java/org/openmrs/module/webservices/rest/web/RestUtil.java [hunk 4]
```diff
@@ -726,7 +726,7 @@ public static boolean hasCause(Throwable throwable, Class<? extends Throwable> c
 				        + ") does not appear to be a valid URL / URI.  Strange, since we got it from the system...", e);
 			}
 			catch (IllegalArgumentException ex) {
-				//ex.printStackTrace();
+
```
Comment (sacull) [inline omod-common/src/main/java/org/openmrs/module/webservices/rest/web/RestUtil.java:729]:
```suggestion
```
Same as above.

**Comment by EllingtonK** [comment]
Sorry, that was sloppy of me not to catch those.  Thanks for noticing.

**Review by sacull — APPROVED** [review]
Without the old code, I think the file looks better. And if someone needs this old code, they will have to learn git. ;)

**Comment by dkayiwa** [comment]
@sacull tru dat! 😊

**Pull request merged**